using-mobile-extensions/adobe-edge-identity/edge-identity-faq.md

```diff
@@ -125,4 +125,184 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 }
 ```
 {% endtab %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+
+## Q: Will an existing Experience Cloud ID (ECID) migrate to the AEP Edge Identity extension?
+### A: Yes
+If the application previously installed the AEP Identity direct solution extension and upgrades to the AEP Edge Identity extension, the existing ECID value is migrated to the AEP Edge Identity extension on first launch of the application.
+
+Note, however, if the Mobile SDK's privacy status was set to `optedOut` at the time the application is upgraded, the AEP Identity direct solution extension will not have an ECID, as it was cleared. In this case, the AEP Edge Identity extension will generate a new ECID.
+
+
+## Q: What is the Experience Cloud ID (ECID) used by the SDK when using both AEP Edge extensions and Adobe Solutions extensions?
+### A: The AEP Edge Identity extension and the AEP Identity direct solution extension each manage their own ECID.
+
+At first launch of the application after upgrading to the AEP Edge Identity extension, the existing ECID from the AEP Identity direct solution extension is migrated to the AEP Edge Identity extension. In this case both extensions will have the same ECID value.
+
+The [resetIdentities](./edge-identity-api-reference#resetIdentities) API regenerates a new ECID used by the AEP Edge Identity extension. This API call does not change the ECID used by the AEP Identity direct solution extension. After calling this API, the ECID used by each identity extension will be different.
+
+Changing the privacy status to `optedOut` will clear the ECID value used by the AEP Identity direct solution extension. Changing the privacy status back to `optedIn` will generate a new ECID used by the AEP Identity direct solution extension. Privacy status changes do not change the ECID used by the AEP Edge Identity extension. Changing the privacy status will cause the ECID used by each identity extension to be different.
+
+When each identity extension has a different ECID, the AEP Edge Identity extension will include the AEP Identity direct solution ECID in its [IdentityMap](./edge-identity-api-reference#identitymap) by listening to state changes from the AEP Identity direct solution extension. By including the both ECIDs in the `IdentityMap`, the Adobe Experience Platform Identity Service will link the the two ECIDs in the customer's Identity Graph.
```

Comment:
the the two ECIDs > the two ECIDs

Reply:
it might be confusing to include details about shared state changes. let's simplify this a bit

When each identity extension has a different ECID, the AEP Edge Identity extension will include the AEP Identity direct solution ECID in its [IdentityMap](./edge-identity-api-reference#identitymap), and so the Adobe Experience Platform Identity Service will link the the two ECIDs in the customer's Identity Graph.